pwiz_tools/commandline/msconvert.cpp

```diff
@@ -841,8 +841,13 @@ void processFile(const string& filename, const Config& config, const ReaderList&
         MSData& msd = *msdList[i];
         try
         {
+            
             // calculate SHA1 checksums
+            
+            *os_ << "calculating source file checksums... ";
```

Comment:
Let's just go with `*os_ << "calculating source file checksums" << endl;`
We don't do the "doing" ... "done" thing on any other progress message.

However, maybe you're adding the "done" message because the next step is wrapping the spectrum list, and if those ctors take a long time it can look like it's spending all that time hashing. I recently fixed this by adding IterationListRegistry to the wrapping. If there's a long running wrapper (like SpectrumList_Filter, or SpectrumList_Demux), it should send progress messages to the iteration listener. That will update the output and let the user know what's going on. In both msconvert.cpp and the GUI.

Reply:
Yep, the SpectrumList_Demux wrapper initialization can be time consuming so I added that done message for that reason. I'll remove that.

Reply:
fixed in 2deaf41